spyder/utils/qthelpers.py

```diff
@@ -562,13 +582,14 @@ def create_plugin_layout(tools_layout, main_widget=None):
     """
     layout = QVBoxLayout()
     layout.setContentsMargins(0, 0, 0, 0)
-    spacing = calc_tools_spacing(tools_layout)
-    if spacing is not None:
-        layout.setSpacing(spacing)
-
-    layout.addLayout(tools_layout)
+    layout.setSpacing(0)
+    try:

```

Comment:
Why is this needed? perhaps a comment would be nice :-)

Reply:
Just to be sure, are you talking about the `layout.setSpacing(0)` or the `try` block? Thanks.

Reply:
(the try!)

Reply:
I added a comment to explain this.

Maybe we can also raise a deprecated warning to warn maintainers of external plugins of this change?